usr/share/rear/restore/NFS_SERVER/default/400_restore_with_nfs_server.sh

```diff
@@ -0,0 +1,25 @@
+# 400_restore_with_nfs_server.sh
+
+FILE=$TARGET_FS_ROOT/rear_finished.txt
+
+NFS_PORT=2049
```

Comment:
If hard-coded then this should be used also for starting the NFS server, or maybe make it a configuration option that defaults to 2049

Reply:
Let's hard-code this in the `ss` call as this is the standard port and as we don't make the port a configuration item